docs/auth.md

```diff
@@ -0,0 +1,15 @@
+# Authentication and Authorisation
+
+All Cortex components take the tenant ID from a header `X-Scope-OrgID`
+on each request. They trust this value completely: if you need to
+protect your Cortex installation from accidental or malicious calls
+then you must add an additional layer of protection.
+
+Typically this means you run Cortex behind a reverse proxy, and ensure
+that all callers, both machines sending data over the remote_write
+interface and humans sending queries from GUIs, supply credentials
+which identify them and confirm they are authorised.
+
+When configuring the remote_write API in Prometheus there is no way to
```

Comment:
Implicitly you can define headers in the remote write API in prometheus by setting a bearer token (which then will be used in the `Authorization` header). This is what we use and we also embed the tenant id inside of it and sign the token on our cortex gateway.

Reply:
Is that different to "http user and password" ?

Reply:
username/password and bearer tokens are just different forms of data in the Authorization header. I guess to be complete you could say "The bearer_token or username and password fields can be set to convey..."